### PR TITLE
Start page UI

### PR DESCRIPTION
**Pull request by fishyvishy**
Merged Ricky's fixes into branch. When merged into main, everyone should be able to create users, log in, and do a filtered search for recipes.

### CODE REVIEW

File: src/main/java/UI/searchPageHelpFrame.java [hunk 1]
```diff
@@ -76,18 +80,18 @@ public searchPageHelpFrame(UserEnterIngredientsController ingredientsController,
 
         //creates filter comboBoxes and add it to panel3
         String[] mealStr = {"No meal filter", "Breakfast", "Brunch", "Lunch/dinner", "Snack", "Teatime"};
-        mealType = new JComboBox<>(mealStr);
-        mealType.addActionListener(e -> System.out.println(mealType.getSelectedItem()));
+        mealType = new JComboBox<String>(mealStr);
+//        mealType.addActionListener(e -> mealTypeFilter = (mealType.getSelectedItem()));
```
Comment (EricKarpovits) [inline src/main/java/UI/searchPageHelpFrame.java:85]: Don't add commented code to main unless it might be used later.

**Review by EricKarpovits — APPROVED** [review]
Great patch @fishyvishy

**Review by takeachangs — CHANGES_REQUESTED** [review]
dont forget to docstring everything!

**Review by Shaffaan — APPROVED** [review]

File: src/main/java/UI/searchPageHelpFrame.java (hunk 1 above)
Comment (fishyvishy) [reply src/main/java/UI/searchPageHelpFrame.java:85]: This is part of Jacky's code @Zhuofan-Y, I'm not sure why it's there.

**Review by takeachangs — APPROVED** [review]

**Pull request merged**